### PR TITLE
libfstrm: fix formality issues

### LINKED ISSUE #9022: libfstrm: missing URL and TITLE is too long
Maintainer: @James-TR

Description:

Recently added package #8983 has formal issues.

- Missing URL
- Mixing tabs and 2 spaces. Use 2 spaces in the description instead of tabs.
- PKG_TITLE is too long and that's why the title isn't shown in make menuconfig
image

Not sure, but isn't there missing dependency, which has @ja-pa in his [Makefile](https://gitlab.labs.nic.cz/turris/turris-os-packages/commit/953d0ed46e9c0769e6ed53b1759bde2984356c71) in his dev branch?

### PR DESCRIPTION
**Pull request by James-TR**
Maintainer: me

Correct issues with TITLE and add URL pointing to the fstrm GitHub Page. Reformat the description so it renders correctly in menuconfig.

Fixes #9022

Signed-off-by: James Taylor <james@jtaylor.id.au>

### CODE REVIEW

File: libs/libfstrm/Makefile [hunk 1]
```diff
@@ -21,18 +21,19 @@ PKG_INSTALL:=1
 include $(INCLUDE_DIR)/package.mk
 
 define Package/libfstrm
-  SECTION:=libs
-  CATEGORY:=Libraries
-  TITLE:=This is fstrm, a C implementation of the Frame Streams data transport protocol.
+	SECTION:=libs
```
Comment (neheb) [inline libs/libfstrm/Makefile]: Normal indentation is 2 spaces.

Comment (James-TR) [reply libs/libfstrm/Makefile]: I've removed those changes

**Comment by James-TR** [comment]
Should the description have some spaces to indent it as well? I'm not sure on how it looks without any indentation.

**Comment by BKPepe** [comment]
Ideally it should have. Remove dot in PKG_TITLE and it's good from my side.

**Comment by James-TR** [comment]
@BKPepe thanks for the feedback! I've removed the dot and added two spaces indentation into the description after looking at other Makefiles for comparison.

**Pull request merged**